### ISSUE TITLE
How do you add apps to WIP that have no publisher?

### ISSUE THREAD

**Issue by domcote** [MicrosoftDocs/windows-itpro-docs#897]
Is that even supported? We have scanner software that apparently does not have a publisher. AppLocker shows the publisher as <blank>.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7df27add-73fe-4f0b-7e30-face0f95cda3
* Version Independent ID: 3ec647d9-3470-8d8b-a941-d56f6522a35e
* Content: [Create a Windows Information Protection (WIP) policy with MDM using the Azure portal for Microsoft Intune (Windows 10)](https://docs.microsoft.com/en-us/windows/security/information-protection/windows-information-protection/create-wip-policy-using-intune-azure)
* Content Source: [windows/security/information-protection/windows-information-protection/create-wip-policy-using-intune-azure.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/security/information-protection/windows-information-protection/create-wip-policy-using-intune-azure.md)
* Service: **w10**
* GitHub Login: @eross-msft
* Microsoft Alias: **justinha**

**Comment by Justinha** [comment]
@domcote yes I think you can use Get-AppLockerFileInformation -Path "<path of the exe>"

**Comment by Justinha** [comment]
I mean:
Get-AppLockerFileInformation -Path "<path of the exe>"

**Comment by Justinha** [comment]
That should the path to the exe in the quotation marks

**Comment by gobimages** [comment]
where in intune do you add the AllowAzureRMSForEDP for AIP with windows information protection?

**Comment by Justinha** [comment]
@domcote sorry for delay here. I added steps under "To create an Executable rule and xml file for unsigned apps".
